librz/core/cmd_debug.c

```diff
@@ -1443,24 +1443,145 @@ static bool get_bin_info(RzCore *core, const char *file, ut64 baseaddr, PJ *pj,
 	return true;
 }
 
+// dm
+RZ_IPI RzCmdStatus rz_cmd_debug_list_maps_handler(RzCore *core, int argc, const char **input, RzOutputMode mode) {
```

Comment:
I wonder if it's better to use `argv` instead of `input` as it's done in other places, to keep consistency. cc @ret2libc

Reply:
yeah i will change that to argv